charts/orchestrator/values-rc.yaml

```diff
@@ -0,0 +1,22 @@
+isReleaseCandidate: true # Indicates RC builds should be used by the chart to install RHDH and Sonataflow
+ocpVersion: v4.15 # The version of the target OCP cluster. Supported versions are: v4.13, v4.14, v4.15, v4.16
+
+sonataFlowOperator:
+  enabled: true # whether the operator should be deployed by the chart
+  subscription:
+    namespace: openshift-serverless-logic # namespace where the operator should be deployed
+    channel: alpha # channel of an operator package to subscribe to
+    installPlanApproval: Automatic # whether the update should be installed automatically
```

Comment:
I'm concerned that if the user sets `Manual` then the chart will fail to deploy unless he interacts with the cluster to approve the deployment before helm gives up. For the operator that's a different story but still I'm not sure we want the user to mingle with the value of the deployment. Perhaps the better approach is to allow the user to deploy their own custom subscription instead, in which case we bypass deployment of ours. WDYT?